test.py

```diff
@@ -0,0 +1,58 @@
+import requests
+import unittest
+
+
+class TestServerFunctionality(unittest.TestCase):
+
+	def setUp(self):
+		self.url= 'http://ec2-52-11-248-121.us-west-2.compute.amazonaws.com/api/v1'
```

Comment:
def setUp(self): 
It will be executed before each test, so it's not correct place for constant
move it to
+class TestServerFunctionality(unittest.TestCase):
     BASE_URL = 'http://ec2-52-11-248-121.us-west-2.compute.amazonaws.com/api/v1'

also yo can move it to another class BASE_URL vill be used in different places of code